src/lib/templates.js

```diff
@@ -67,7 +71,9 @@ const templates = {
   `,
   footer: `
     <div class="${footerPrefix}">
-      Search by <a class="${footerPrefix}--logo" href="https://www.algolia.com/docsearch">Algolia</a>
+      <a class="${footerPrefix}--logo" aria-label="Search by algolia" href="https://www.algolia.com/docsearch">
+        <img src="${searchByAddress}" alt="Search by algolia" height="100%" width="100%">
```

Comment:
I don't think we should be using an external url for the image. I would rather have it either as an inline SVG in the JavaScript, or as a background image from the CSS. In any case, if we use an URL, we should use one that targets an origin we control (ie. something in the DocSearch repo, or DocSearch website, not on the main Algolia website).

I'm gonna ping @JonasBa on this: How do you suggest we include the "Powered by Algolia" logo in DocSearch? As an inline SVG, a background image or a hotlink?

Reply:
I will use https://instantsearch-css.netlify.com/widgets/powered-by/ instead

Reply:
Oh neat.